docker-dev-setup/README.md

```diff
@@ -62,6 +62,9 @@ docker exec -it mars_dev bash -c "php artisan serve --host=0.0.0.0"
 
 The app is now running at [http://localhost:8000](http://localhost:8000).
 
+You can log in as `example@eotvos.elte.hu` to gain access to a superuser account or as `collegist@eotvos.elte.hu` for a regular account.
```

Comment:
These values can be configurable in `.env`. This is already stated at https://github.com/EotvosCollegium/mars/blob/development/_SETUP_DEV_ENVIRONMENT.md#for-everyday-use but you can put a link here pointing to this one.